### PR TITLE
Organization settings

### PR DESCRIPTION
**Pull request by amasciulli**
Add settings for organization and add framgent in onboarding process to specify organization

/!\ Don't merge before #4

### CODE REVIEW

File: app/src/main/java/com/genymobile/pr/ui/ChooseOrganizationFragment.java [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+    private void saveOrganization() {
+        SharedPreferences preferences = PreferenceManager.getDefaultSharedPreferences(getActivity());
+        preferences.edit()
+                .putString(getString(R.string.pref_organization), organizationText.getText().toString())
+                .commit();
+    }
+
+    private void showPullRequests() {
+        fragmentCallbacks.onOrganzationChosen();
```
Comment (genypaul) [inline app/src/main/java/com/genymobile/pr/ui/ChooseOrganizationFragment.java]: typo :wink:

File: app/src/main/java/com/genymobile/pr/ui/PullRequestDetailsDialogFragment.java [hunk 2]
```diff
@@ -55,7 +58,7 @@ private void getIssue() {
         String login = preferences.getString(getString(R.string.pref_login), null);
         String password = preferences.getString(getString(R.string.pref_password), null);
         GitHubProvider provider = new GitHubProvider(login, password);
-        provider.getIssue("Genymobile", repo, number).enqueue(new IssueCallback());  //TODO owner will be in pref in a future PR
+        provider.getIssue(organization, repo, number).enqueue(new IssueCallback());
```
Comment (genypaul) [inline app/src/main/java/com/genymobile/pr/ui/PullRequestDetailsDialogFragment.java:61]: :+1:

File: app/src/main/java/com/genymobile/pr/ui/PullRequestListFragment.java [hunk 3]
```diff
@@ -29,19 +32,29 @@
 import java.util.List;
 
 public class PullRequestListFragment extends Fragment {
-    private static final String GENYMOBILE = "Genymobile";
-    private static final String PULL_REQUEST_DETAILS_DIALOG_TAG = "pull-request-details-dialog-tag";
+    private static final String TAG = PullRequestListFragment.class.getSimpleName();
+    private static final String PULL_REQUEST_DETAILS_DIALOG_TAG = "pull_request_details_dialog_tag";
```
Comment (genypaul) [inline app/src/main/java/com/genymobile/pr/ui/PullRequestListFragment.java:36]: :+1:

File: app/src/main/java/com/genymobile/pr/ui/MainActivity.java [hunk 4]
```diff
@@ -20,18 +17,30 @@ protected void onCreate(Bundle savedInstanceState) {
...
-                preferences.contains(getString(R.string.pref_password))) {
-            fragment = new PullRequestListFragment();
+
+        boolean credentialsSet = preferences.contains(getString(R.string.pref_login)) &&
+                preferences.contains(getString(R.string.pref_password));
+        boolean organizationSet = preferences.contains(getString(R.string.pref_organization));
+
+        if (credentialsSet) {
+            if (organizationSet) {
+                fragment = new PullRequestListFragment();
```
Comment (genypaul) [inline app/src/main/java/com/genymobile/pr/ui/MainActivity.java:27]: [minor] Just thinking, in a future PR, we should rename this to have PullRequest**s**Fragment (plural)

Comment (amasciulli) [inline app/src/main/java/com/genymobile/pr/ui/MainActivity.java:27]: I never know how to call them, because both are grammatically correct (and plural)

Comment (genypaul) [inline app/src/main/java/com/genymobile/pr/ui/MainActivity.java:27]: I dont know either. I just thought "this one is long", lets shorten it a bit :)

Comment (amasciulli) [inline app/src/main/java/com/genymobile/pr/ui/MainActivity.java:27]: PullRequestsFragment maybe ?

Comment (genypaul) [inline app/src/main/java/com/genymobile/pr/ui/MainActivity.java:27]: That is what i suggested in the first place :)

**Comment by genypaul** [comment]
LGTM

**Pull request merged**